fractal_server/utils.py

```diff
@@ -82,6 +110,8 @@ async def execute_command(*, cwd: Path, command: str) -> str:
         cwd=cwd,
     )
     stdout, stderr = await proc.communicate()
+    logger = logging.getLogger(logger_name)
```

Comment:
Isn't `logger` the same that you already obtained with `set_logger`?

Reply:
you are right. will fix